index.html

```diff
@@ -30,16 +48,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
```

Comment:
Unless there is no way around it, try to keep all CSS in one place, like your external CSS file.  This is given precedence over anything else except for !important.  So if you wanted to change to a different color if the user mouse hovered over the box then these line styles would prevent that from happening.